src/Uno.UI.Runtime.Skia.Gtk/GtkHost.cs

```diff
@@ -137,6 +141,23 @@ public void Run()
 			Gtk.Application.Run();
 		}
 
+		private bool InitializeGtk()
+		{
+			try
+			{
+				Gtk.Application.Init();
+				return true;
+			}
+			catch(System.TypeInitializationException e)
```

Comment:
There is `using System;` in the file. Is there an ambiguity?
Also fixing formatting to reduce noise for the formatting PR :)

```suggestion
			catch (TypeInitializationException e)
```

Reply:
No particular reason, this was a copy/paste of the exception type in the console :)